software-engineer-in-test-US.md

```diff
@@ -0,0 +1,83 @@
+We are aiming to be a cross-functional team where Quality Engineers help developers to build quality products.
+
+### You will
+- Understand Banked products, their features and how users interact with them and develop scalable and reliable automated testing frameworks for testing Banked's software. This includes end-to-end web/api tests, service level tests, integration tests, contract tests, performance tests, infrastructure tests, chaos testing, security tests and exploratory tests.
+
+- Come up with Acceptance Test Plan during new market adoption and drive the test programme with third-party suppliers.
+
+- Help engineers to own quality and help improve the quality processes through training, mentoring and pair programming.
+
+- Engage with Product Management on the feasibility of requested features, clarifying requirements where necessary and identifying challenges.
+
+- Contributes to technical design, thinking through failure cases and building quality from the earliest phase of SDLC. (Shift left thinking)
+
+- Help engineers to build quality rather than just testing.
+
+- Take ownership and work autonomously on any development task within an application or service, delivering code on time and with a quality mindset.
+
+### You may be a good fit if you have
+- Significant experience building test automation solutions, automated testing frameworks, monitoring systems and improving test effectiveness.
+Experience working with public cloud infrastructure and services.
+
+- Experience in all or a subset of: 
+    * Automated end-to-end testing of web applications.
+    * Automated end-to-end testing of native mobile applications Automated testing and verification of HTTP APIs.
+    * Automated performance, load and soak testing of user interfaces or APIs. 
+    * Automated infrastructure testing or chaos testing.
+    * Automated security and vulnerability testing.
+    * Ability to determine the right open-source/tool to address tooling requirements.
+    * Ability to write and understand unit tests. 
+    * Know or are willing to learn "go" programming language. 
+    * Basic knowledge of mobile app test automation.
+    * Practical experience building quality for microservices.
+    * Practical experience building quality for distributed systems containing several microservices.
+    * Understanding of how best practises with Quality Engineering.
+    * Understanding of technologies like traceability, cloud computing and Kubernetes.<BR>
+
+- Experience working with production-grade JavaScript.
+- Significant experience in communicating and working with software teams, and working towards building inclusive, fun environments.
+- Experience with the US Banking domain is preferred but not a mandatory condition.
+- A passion for learning and growth, picking up new skills and tools.
+- Solid knowledge on front-end technologies.
+- A track record of developing system maintainability and ease of operation.
```

Comment:
Maybe this needs rewording to something like `A track record of creating maintainable systems with ease of operation` but I'm not 100% sure what we're trying to say here.
Are we asking for people who have built maintainable systems? Or are we asking for someone who advocates for system maintainability?